js/widgets/forms/select.custom.js

```diff
@@ -79,16 +94,24 @@ return $.widget( "mobile.selectmenu", $.mobile.selectmenu, {
 		}
 
 		if ( event.type === "vclick" ||
-				event.keyCode && ( event.keyCode === $.mobile.keyCode.ENTER || event.keyCode === $.mobile.keyCode.SPACE ) ) {
+				event.keyCode &&
+					( event.keyCode === $.mobile.keyCode.ENTER ||
+						event.keyCode === $.mobile.keyCode.SPACE ) ) {
 
 			this._decideFormat();
 			if ( this.menuType === "overlay" ) {
-				this.button.attr( "href", "#" + this.popupId ).attr( "data-" + ( $.mobile.ns || "" ) + "rel", "popup" );
+				this.button
+					.attr( "href", "#" + this.popupId )
+					.attr( "data-" + ( $.mobile.ns || "" ) + "rel", "popup" );
```

Comment:
ns is deprecated move determining this to backcompat and make an extension point